Vagrantfile

```diff
@@ -1,7 +1,23 @@
+db_file_exists    = "test -f /vagrant/edison/db.sql"
+restore_db        = "/vagrant/restore_db_data.sh"
+db_restored_msg   = "echo \"Database restored.\""
+db_not_exists_msg = "echo \"db.sql not exists.\""
+try_restore_db    = "bash -c '#{db_file_exists} && #{restore_db} && #{db_restored_msg} || #{db_not_exists_msg}'"
+
 Vagrant.configure("2") do |config|
+  config.trigger.before :destroy do |trigger|
+    trigger.info = "Saving database data inside synced folder"
+    trigger.run_remote = {inline: "/vagrant/save_db_data.sh"}
```

Comment:
to be consistent, shouldn't this file be declared as a variable on the top?

Reply:
Yes, on it.

Reply:
I squashed some commits so this file is no longer reachable through this commit.
So I'm resolving this conversation.